### ISSUE TITLE
WakaTime asks for API key every start of VS15Preview

### ISSUE THREAD

**Issue by jakubsuchybio** [wakatime/visualstudio-wakatime#58]
Hi,
so I have installed VS2015Update2 and also VS15Preview.
When I try anything (delete .wakatime.* in ~; uninstall and install wakatime extension; restart VS2015Update2) VS2015Update2 works like a charm.
But when I try all of mentioned above in VS15Preview, every time I start VS15Preview I get window to enter API key, where there is my API key already inserted. See here:
![image](https://cloud.githubusercontent.com/assets/6567287/14388402/3b7435dc-fdae-11e5-8289-1cba9ad206ce.png)
I tried to enable debugging, if something would show in the .wakatime.log, but there is nothing from inicialization of key. Here is the Wakatime output from Output window in VS15Preview:
```
[Wakatime Info 05:20:28 PM] Please input your api key into the wakatime window.
[Wakatime Info 05:20:28 PM] Initializing WakaTime v7.0.2
[Wakatime Debug 05:20:28 PM] Python found from embedded location: C:\Users\Jakub\python\pythonw
[Wakatime Info 05:20:28 PM] Current wakatime-cli version is 4.1.13
[Wakatime Info 05:20:28 PM] Checking for updates to wakatime-cli...
[Wakatime Debug 05:20:28 PM] No proxy will be used. It's either not set or badly formatted.
[Wakatime Info 05:20:29 PM] wakatime-cli is up to date.
[Wakatime Info 05:20:29 PM] Finished initializing WakaTime v7.0.2
```
If you need any more information, I am happy to help.

Edit: So it started to asking for API key in VS2015Update2 also :/

**Comment by alanhamlett** [comment]
Fixed with 64349f05139346e064cff2c1aec64bcf5d8623d2.

**Comment by jakubsuchybio** [comment]
Thank you. Works like a charm.
